modules/client/src/multisig/internal/utils.ts

```diff
@@ -67,6 +67,7 @@ export function toMultisigProposal(
     approvals: proposal.approvers.map(
       (approver) => approver.id.slice(0, 42),
     ),
+    potentiallyExecutable: proposal.potentiallyExecutable,
```

Comment:
Why do we need to export this parameter? What cannot be done using the existing fields?